.github/scripts/conventional-commit-pr-title-reminder/utils.js

```diff
@@ -0,0 +1,68 @@
+exports.updateComment = updateComment;
+
+/** @param {import('@types/github-script').AsyncFunctionArguments} AsyncFunctionArguments */
```

Comment:
Unfortunately without much tinkering, we can't install this type in package.json since it enforces node v20. I'll leave the type here since it will help when we need to visit this util again. We only need to install the type again, but not commit the `package.json` and `yarn.lock`